Gemfile

```diff
@@ -18,6 +18,7 @@ gem 'gibberish'
 gem 'octokit'
 gem 'delayed_job_active_record'
 gem 'pundit'
+gem 'honeybadger', require: false
```

Comment:
I suspect honeybadger has to be placed at the bottom of the Gemfile for [reasons](https://github.com/degica/hats/blob/423c057916fa9d814780503e276b741922e93a99/Gemfile#L356-L357).

Reply:
@davidsiaw I think this is fine since we are not including the gem yet `require: false`. The gem gets loaded when Rails initializer is booted... maybe this is okay?

Reply:
Great. That is why. Thanks.